webapp/src/tutorial.tsx

```diff
@@ -651,4 +655,31 @@ export class ChooseRecipeDialog extends data.Component<ISettingsProps, ChooseRec
             </sui.Modal>
         )
     }
+}
+
+export class WorkspaceHeader extends data.Component<any, {}> {
+    private flyoutWidth: number = 0;
+    constructor(props: any) {
+        super(props);
+    }
+
+    componentWillUpdate() {
+        let flyout = document.querySelector('.blocklyFlyout');
+        if (flyout) {
+            this.flyoutWidth = flyout.clientWidth;
+        }
+    }
+
+    private headerStyle() {
+        return {
+            width: this.flyoutWidth
+        }
+    }
+
+    renderCore() {
+        return <div id="headers">
+                    <div id="flyoutHeader" style={this.headerStyle()}>Toolbox</div>
+                    <div id="workspaceHeader">Workspace</div>
```

Comment:
`lf`'s here, or are these intentionally being left in english like `Blocks` & `JavaScript` toggles are?